### DISCUSSION TITLE
docs/v1.11.x/guide/setup-options/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2829] [Announcements]
# docs/v1.11.x/guide/setup-options/

https://kubeovn.github.io/docs/v1.11.x/guide/setup-options/

**Comment by Git4Mark** [comment]
underlay模式使用逻辑网关的情况下，mtu该如何设置呢，默认的1400好像会限制svc请求数据大小，而且这个值不能设置成1500，设置成1500会导致请求pod数据大小限制

**Comment by xinsir88** [comment]
大佬好，因为我们公司内网没有互联网，所有想把咱们得帮助文档搬到我们内网，我该如何操作呢？
